domino-ui/src/main/java/org/dominokit/domino/ui/forms/BasicFormElement.java

```diff
@@ -198,113 +199,113 @@ public T clearInvalid() {
+            getLabelElement().removeChild(requiredIndicator);
+        }
+        
         if (required) {
             addValidator(requiredValidator);
-            Node requiredIndicator = DominoFields.INSTANCE.getRequiredIndicator().get();
+            requiredIndicator = DominoFields.INSTANCE.getRequiredIndicator().get();
             getLabelElement().appendChild(requiredIndicator);
-            this.requiredIndicator = requiredIndicator;
         } else {
             removeValidator(requiredValidator);
-            if (getLabelElement().contains(this.requiredIndicator)) {
-                getLabelElement().removeChild(this.requiredIndicator);
-            }
         }
         return (T) this;
     }
-
+    
     @Override
     public T setRequired(boolean required, String message) {
         setRequired(required);
         setRequiredErrorMessage(message);
         return (T) this;
     }
-
+    
     @Override
     public boolean isRequired() {
         return hasValidator(requiredValidator);
     }
-
+    
     @Override
     public T setRequiredErrorMessage(String requiredErrorMessage) {
         this.requiredErrorMessage = requiredErrorMessage;
         return (T) this;
     }
-
+    
     @Override
     public String getRequiredErrorMessage() {
         return isNull(requiredErrorMessage) ? "* This field is required." : requiredErrorMessage;
     }
-
+    
     @Override
     public T groupBy(FieldsGrouping fieldsGrouping) {
         fieldsGrouping.addFormElement(this);
         return (T) this;
     }
-
+    
     @Override
     public T ungroup(FieldsGrouping fieldsGrouping) {
         fieldsGrouping.removeFormElement(this);
         return (T) this;
     }
-
+    
     @Override
     public void setValue(V value) {
         value(value);
     }
-
+    
     public boolean isFixErrorsPosition() {
         return fixErrorsPosition;
     }
-
+    
     public T setFixErrorsPosition(boolean fixErrorsPosition) {
         this.fixErrorsPosition = fixErrorsPosition;
         return (T) this;
     }
-
+    
     protected abstract DominoElement<HTMLDivElement> getFieldInputContainer();
-
+    
     protected abstract DominoElement<HTMLDivElement> getFieldContainer();
-
+    
     protected abstract DominoElement<HTMLElement> getHelperContainer();
-
+    
     protected abstract DominoElement<HTMLElement> getErrorsContainer();
-
+    
     protected abstract DominoElement<HTMLLabelElement> getLabelElement();
-
+    
     public DominoElement<HTMLElement> getLabelTextElement() {
         return DominoElement.of(getLabelElement().element());
     }
-
+    
     @Override
     public void showErrors(List<EditorError> errors) {
         List<String> editorErrors = errors.stream()
-                .filter(e -> this.equals(e.getEditor()))
-                .map(EditorError::getMessage)
-                .collect(Collectors.toList());
-
+                                          .filter(e -> this.equals(e.getEditor()))
```

Comment:
Maybe there is a formatting issue, there is a lot of space changes

Reply:
Yes, copy & paste issue.
Will fix it.